js/packages/components/chat/message/index.ts

```diff
@@ -0,0 +1,6 @@
+export { Message } from './Message'
```

Comment:
I wouldn't expose all the message types here but only the polymorphic message component here, maybe even `mv Message.tsx index.ts`

if someone want only one message it can import the file of the message